ui/Makefile

```diff
@@ -143,3 +143,7 @@ $(GOBINDATA_DEBUG_TARGET): $(GOBINDATA_DEBUG_DEPS) bower.installed debug/$(INDEX
 
 watch: debug
 	gulp watch
+
+.PHONY:
```

Comment:
I don't think `.PHONY:` on its own does anything. It should be `.PHONY: clean` (see the top-level makefile)